### ISSUE TITLE
Allow user to override any picture

### ISSUE THREAD

**Issue by phbernard** [RealFaviconGenerator/realfavicongenerator#53]
RealFaviconGenerator generates a lot of pictures automatically, which is exactly what it is supposed to do.

However, some users might want to override certain pictures. In particular, the 16x16 and 32x32 PNG/ICO pictures are good candidates for manual edition. In the options, the user should be able to override any picture:
- favicon.ico, 16x16 picture
- favicon.ico, 32x32 picture
- favicon.ico, 48x48 picture
- PNG icon, 16x16
- PNG icon, 32x32
- ...
- Apple Touch icon, 57x57
- ..

(or maybe only 16x16 and 32x32 pictures are enough, at least for a first shot? To be decided the day I address this issue)

**Comment by jacksonj04** [comment]
I'd like to be able to override the 'small' sizes (to me that's 32px and down) since those are the ones where I'm most likely to want to reduce detail. Larger sizes seem to work well anyway.

Maybe it's worth initially just offering an option for "low detail image" which is only used for these small sizes?

**Comment by phbernard** [comment]
Yeah 16x16 and 32x32 seem to be the real candidates for override. Thank you for the feedback.

**Comment by smaddock** [comment]
+1 on this. We've edited the 16x16 to be clearer; replacing the PNGs is no problem but it would be nice if that could be specified in RFG instead of having to rebuild the .ico.

**Comment by giovannipds** [comment]
+1, I want the same feature. I also think the user should have the option to specify other image for smaller favicons.

**Comment by indolering** [comment]
@giovannipds Don't do "+1" posts, that is what reaction buttons are for.  I like a clean Github notification feed.

**Comment by giovannipds** [comment]
@indolering Excuse me, I wasn't the only one, @smaddock did the same, even before me.
And for your understanding, on mobile we don't have GitHub's reactions.
Although, I don't really care what you like or not, so sorry. 👍

**Comment by hudri** [comment]
Shameless +1 bump due Google's new behavior of showing desktop fav icons in their search results. Please add the "dedicated picture" tab for Desktop browsers.

In our agency the common workflow is to create two separate base images: one large, detailled image used to generate all mobile/retina icons, and one small, reduced image used to generate only the 16/32px desktop browser icons.

Right now this is possible, but a bit cumbersome, because I'd need to upload the desktop icon as main image, and then separatly override all other versions.
